smartgrid-prediction/diagrams.graffle

```diff
@@ -0,0 +1,653 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
would be nice if the line for "rebuild model" went the other direction (around Redis and pointing to predict from below) so it would not cross any other lines.